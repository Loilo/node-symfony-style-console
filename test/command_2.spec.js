

const wrapper = require('./wrapper')

test('02 Ensure has single blank line between blocks', wrapper(io => {
	io.warning('Warning')
	io.caution('Caution')
	io.error('Error')
	io.success('Success')
	io.note('Note')
	io.block('Custom block', 'CUSTOM', 'fg=white;bg=green', 'X ', true)
}))
