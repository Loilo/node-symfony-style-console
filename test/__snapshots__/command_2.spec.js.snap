// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`02 Ensure has single blank line between blocks 1`] = `
"[WARNING] Warning                                                              

 ! [CAUTION] Caution                                                            

 [ERROR] Error                                                                  

 [OK] Success                                                                   

 ! [NOTE] Note                                                                  

X [CUSTOM] Custom block"
`;
